README.md

```diff
@@ -567,6 +567,12 @@ Returns `value` padded to the length specified by `to_size` with the string `pad
 'test12312'
 ```
 
+#### `zpad_bytes(to_size, value)`
+
+A convenience method, easily curry-able for passing a value through a pipeline of formatters.
+
+Equivalent to `pad_left(value, to_size, b'\x00')`, but accepting only `bytes` arguments.
+
```

Comment:
Given that pretty much every single other section has it's own example section I think this one should too.   Since the `left_pad` and `right_pad` functions may go away in favor of `left_pad_text` and `right_pad_text` I think it's best to not even reference `left_pad` and `right_pad`.